graphql_query_examples/graphql_query_examples.md

```diff
@@ -1200,6 +1200,50 @@ query get_results_for_list_of_checks {
 ```
 </details>
 
+### 🔎 services > query for all services with openapi docs
+
+If an OpenAPI doc is found for a service, the `preferredApiDocumentSource` 
+field will be set. Otherwise `preferredApiDocumentSource` will be null.
+
+```graphql
+query all_services_with_openapi_docs($endCursor: String) {
+  account {
+    services(after: $endCursor) {
+      nodes {
+        name
+        preferredApiDocumentSource
```

Comment:
```suggestion
        preferredApiDocument
```

Because a service can specify that `pull` is the preferred source but there could be no doc in the repo

Reply:
oh true - i assumed this was just checking that a preferred source was set - this makes more sense :)

Reply:
Thanks, this definitely makes more sense. However, it seems like I have services returning valid ids for preferredApiDocument, but the content is null. So I'll do a check for both id and content are null.

Reply:
we should just check if `content` is null